common/library/module_utils/local_repo/config.py

```diff
@@ -54,7 +54,7 @@
 RHEL_OS_URL = "rhel_os_url"
 SOFTWARES_KEY = "softwares"
 USER_REPO_URL = "user_repo_url"
-VAULT_KEY_PATH = "/opt/omnia/input/project_default/.omnia_config_credentials_key"
+VAULT_KEY_PATH = "/opt/omnia/input/project_default/.local_repo_credentials_key"
```

Comment:
This is hard coded path, better take it from input , becuase this path can change -